estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,59 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api, exceptions
+from datetime import date, timedelta
+
+
+class EstatePropertyOffer(models.Model):
+    _name = 'estate.property.offer'
+    _description = 'Estate Property Offer'
+    _order = "price desc"
+
+    price = fields.Float(string='Price')
+    status = fields.Selection(string='Status', copy=False,
+                            selection=[('accepted', 'Accepted'), ('refused', 'Refused')])
```

Comment:
Same, this indentation looks a little bit strange. When you have multiline arguments IMO it's better to do one argument per line
```suggestion
    status = fields.Selection(
        string='Status', 
        copy=False,
        selection=[('accepted', 'Accepted'), ('refused', 'Refused')]
    )
```